processing/src/test/java/org/apache/druid/query/topn/TopNQueryRunnerBenchmark.java

```diff
@@ -46,6 +46,7 @@
 /**
  * Based on TopNQueryRunnerTest
  */
+@Ignore
 public class TopNQueryRunnerBenchmark extends AbstractBenchmark
```

Comment:
I think this file was misplaced; it should be in the `benchmark` module

Reply:
Yeah, we have a few other similar files as well that need to be moved to benchmarks module.